src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -2075,7 +2086,179 @@ let rectifyTermColumns () =
         }
     )
 
-// Old stuff, mostly deprecated
+/// <summary>
+/// Tries to get the name of the top level excel worksheet of top level metadata
+/// </summary>
+let tryGetTopLevelMetadataSheetName (context:RequestContext) =
+    promise {
+        let worksheets = context.workbook.worksheets
+
+        let _ = worksheets.load(propertyNames = U2.Case2 (ResizeArray[|"values"; "name"|]))
+
+        do! context.sync().``then``(fun _ -> ())
+
+        let worksheetItems = worksheets.items |> Array.ofSeq
+
+        let metadata =
+            worksheetItems
+            |> Array.choose (fun item -> if item.name.Contains("isa_") then Some item else None)
```

Comment:
use ARCtrl logic to check for metadatasheets

Reply:
fixed